includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -394,6 +394,45 @@ private function adjust_and_replace_nodes_in_array_map( $node_lists ) {
 		}
 	}
 
+	/**
+	 * Add 'srcset' attribute to image node if lightbox is enabled for that.
+	 *
+	 * @param Element $node Image node.
+	 *
+	 * @return void
+	 */
+	private function set_srcset_for_lightbox_images( Element $node ) {
+
+		if ( $node->hasAttribute( Attribute::DATA_ID ) ) {
+			$attachment_id = intval( $node->getAttribute( Attribute::DATA_ID ) );
+			$srcset        = wp_get_attachment_image_srcset( $attachment_id, 'full' );
+
+			if ( empty( $srcset ) ) {
+				return;
+			}
+
+			$thumbnail_image = wp_get_attachment_image_url( $attachment_id );
+			$srcset          = $thumbnail_image . ' 150w, ' . $srcset;
```

Comment:
The number 150 shouldn't be hard-coded since the value can be changed on the Media screen:

![Screen Shot 2022-05-02 at 10 11 02](https://user-images.githubusercontent.com/134745/166293093-8b9fa329-5412-44ee-967a-71b3206e5d42.png)

Reply:
Thank you for noticing. I have addressed this in my latest commit.